sample.tex

```diff
@@ -188,17 +194,11 @@
 \mainmatter % starts thesis body
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-\chapter{Introduction}
+\chapter[Introduction]{Introduction\footnote{Here is an example of a citation in a chapter heading.}}
```

Comment:
There are some things about the grad school guidelines that I really don't like :) and this is one of them.  I wonder when it would be necessary to actually put a citation in the heading of a chapter?  I'm guessing this wouldn't be an actual BibTeX citation?

If we do include this, could we have an actual example, instead of just saying that this is an example?  Also, it might be good to have a comment in the code explaining this situation.

Reply:
Yes, silly me. I forgot the actual citation!

Reply:
I have never had to cite an author in a section heading myself, but I did notice there was an open issue concerning this. The latest commit includes an example of this in the "Citation" section, which seemed more appropriate.